server/channels/app/export.go

```diff
@@ -171,7 +184,7 @@ func (a *App) BulkExport(ctx request.CTX, writer io.Writer, outPath string, job
 	profilePictures = append(profilePictures, botPPs...)
 
 	ctx.Logger().Info("Bulk export: exporting posts")
-	attachments, appErr := a.exportAllPosts(ctx, job, writer, opts.IncludeAttachments, opts.IncludeArchivedChannels)
+	attachments, appErr := a.exportAllPosts(ctx, job, writer, opts.IncludeAttachments, opts.IncludeArchivedChannels, teamId)
```

Comment:
Maybe a future improvement to change method signature to accept options as a struct so that we don't need to change signature with newer options.